app/upgrades/v2_5_0/v2_5_0_test.go

```diff
@@ -42,6 +43,20 @@ func (s *Suite) TestUpgrade() {
 		funtoken = evmtest.CreateFunTokenForBankCoin(
 			deps, originalbankMetadata.Base, &s.Suite,
 		)
+
+		// ten holders for testing
+		holders = []gethcommon.Address{
```

Comment:
Because we don't have a MAINNET_STNIBI_HOLDERS() set anymore, we create one just for testing.